src/features/environmentCreate/components/Specification/SpecificationCreate.tsx

```diff
@@ -99,7 +104,7 @@ export const SpecificationCreate = ({ onCreateEnvironment }: any) => {
       <Box>
         {show ? (
           <CodeEditor
-            code={formatCode(channels, requestedPackages)}
+            code={stringify(channels, requestedPackages, variables)}
```

Comment:
See https://github.com/conda-incubator/conda-store-ui/pull/354#discussion_r1444104490 for an explanation.

Reply:
Okay, this was a wrong decision. When creating a new env with a clean DB, this would be just `[]` which is not very user-friendly. I'll revert.